xml/chapter2/section5/subsection3.xml

```diff
@@ -551,8 +553,9 @@ function mul_term_by_all_terms(t1, L) {
     able to handle polynomial operations such as
     <LATEX>
       \[
-      {\left[ (y+1)x^2 +(y^2 +1)x+(y-1)\right]\cdot \left[(y-2)
-      x+(y^3 +7)\right]}
+      \begin{array}{l}
+      {\left[ (y+1)x^2 +(y^2 +1)x+(y-1)\right]\cdot \left[(y-2)x+(y^3 +7)\right]}
```

Comment:
I thought the thin space looked good.

Reply:
Not sure how "thin spaces" disappeared. Is it because we moved to "array" instead of just \[ \]?

Reply:
Where do you think there should be thin spaces?

Reply:
Yes, the explanation is in the email. Let's LaTeX do this and not micro-manage, unless we have to, as in the case of k_1 f(...) and k_2 f(...)